packages/jest-matchers/src/index.js

```diff
@@ -78,9 +84,27 @@ const addMatchers = (matchersObj: MatchersObject): void => {
   Object.assign(global[GLOBAL_MATCHERS_OBJECT_SYMBOL], matchersObj);
 };
 
+const _validateResult = result => {
```

Comment:
:themoreyouknow: I benchmarked the arrow vs regular function in JavaScriptCore and to my surprise, arrow functions are natively faster.

```
normal x 53,987,644 ops/sec ±3.50% (44 runs sampled)
arrow x 57,679,587 ops/sec ±3.08% (58 runs sampled)
```